.github/workflows/govcloud.yml

```diff
@@ -23,20 +23,13 @@ jobs:
         with:
           python-version: 3.10.1
       - name: install dependencies
-        run: python -m pip install --upgrade pip && python -m pip install -r requirements.txt
+        run: python -m pip install --upgrade pip && python -m pip install -r dev-requirements.txt
       - name: build static assets
         run: python build.py
       - name: run unit tests
         run: python -m pip install pytest && python -m pytest .
-      - name: Install Chromium
-        uses: browser-actions/setup-chrome@latest
-      - name: setup Chromedriver
-        uses: nanasess/setup-chromedriver@v1
-        with:
-        # Optional: do not specify to match Chrome's version
-          chromedriver-version: '88.0.4324.96'
       - name: Run Accessability tests
-        run: pytest --driver Chrome --driver-path .venv/lib/python3.10/site-packages/chromedriver_py/chromedriver_linux64
+        run: pytest
```

Comment:
Seems like we're running all tests already above in https://github.com/communitiesuk/funding-service-design-assessment/blob/292e37302cead998cb13001ac637b23ccb0dc8d3/.github/workflows/govcloud.yml#L30 so don't think this is adding any value?